translate/storage/mozilla_lang.py

```diff
@@ -91,7 +91,8 @@ def parse(self, lines):
                 readyTrans = False  # We already have our translation
                 continue
 
-            if line.startswith('#'):  # A comment
+            if line.startswith('#') and not line.startswith('##'):
```

Comment:
Normally comments should start with `# `, but it's not impossible for a comment to miss the space. I think it's safer to exclude `##`.
This is also the same check we do while parsing .lang files in our tools.